### DISCUSSION TITLE
Add new row on top in Quasar Table

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#10283] [General - Components / Directives / etc]
I have developed one Quasar data table with Add , Edit & Delete button . 

When user click add button new row will be added in that table , but that new row is getting displayed in the bottom , can anyone guide me how to show in top because that row contains 3 drop down & 2 text field for data entry , so it will be easy to the user to enter the data instead of scrolling to last row each time.

consider following example  [(https://codepen.io/metalsadman/pen/ZgKexK](url) it this above code we have to add one button call add we user click that button new row should get added in the top of the row instead of bottom

**Comment by jmriyaz84** [comment]
Any update / help ? please guide it is blocker for me

**Reply by ontwikkelfabriek** [reply]
@jmriyaz84 

> Any update / help ? please guide it is blocker for me

You could help yourself getting better support by creating a working codepen ( or modifying the codepen your refer to) that demonstrates your issue. 

btw the codepen link you posted is **not** correct when you click on it....
